lib/phx_tools_web/components/layouts/root.html.heex

```diff
@@ -17,7 +17,14 @@
     >
     </script>
   </head>
-  <body class="bg-white antialiased">
+
+  <body class="font-montserrat bg-[#110B33]">
```

Comment:
Can we put the colors as shades of existing colors?
https://tailwindcss.com/docs/customizing-colors#adding-additional-colors

Reply:
@almirsarajcic I have a few questions with regards to this comment

1. Do you want me to do this for all colors? or just for this one?
2. What name do you want me to use for the color?

Reply:
If we have the same custom color, font size, margin, or something like in multiple places, lets have them in the Tailwind config and then reference them using the given name.

Reply:
I'm trying to do it, but after I moving hard coded value to the tailwind.config and use the reusable one, the interface result is slightly different so I only do it in the new branch.

https://github.com/nittin-shankar/phx.tools/tree/phx_tools_landing_var

Reply:
Thank you @amir-repository. I have looked at your code and found out the why the interface is looking very different. This happens because you have configured it directly to the theme
```
 module.exports = {
   content: ['./js/**/*.js', '../lib/*_web.ex', '../lib/*_web/**/*.*ex'],
   theme: {
     fontSize: {
       '1xs': '0.8125rem' /* 13px */,
       '1sm': '0.938rem' /* 15px */,
       '1xl': '1.375rem' /* 22px */,

       ....
       ....
```

However, it should have been configured to the extend key like below:
```
 module.exports = {
   extend: {
       fontSize: {
          '1xs': '0.8125rem' /* 13px */,
          '1sm': '0.938rem' /* 15px */,
          '1xl': '1.375rem' /* 22px */,
       }
     ....
```

I also noticed that you configured **all** custom arbitrary values in the branch that you were working. We only needed to extend it for custom values that appear more than once. So, I completed the work now within the current branch.

Reply:
Thank you @nittin-shankar, you help me a lot